Makefile

```diff
@@ -202,19 +202,19 @@ test-docker-mount-code:
 	PACKAGES_DIR=$$(echo $$(pwd)/.venv/lib/python*/site-packages | awk '{print $$NF}'); \
 		DOCKER_FLAGS="$(DOCKER_FLAGS) --entrypoint= -v `pwd`/localstack/config.py:/opt/code/localstack/localstack/config.py -v `pwd`/localstack/constants.py:/opt/code/localstack/localstack/constants.py -v `pwd`/localstack/utils:/opt/code/localstack/localstack/utils -v `pwd`/localstack/services:/opt/code/localstack/localstack/services -v `pwd`/localstack/aws:/opt/code/localstack/localstack/aws -v `pwd`/Makefile:/opt/code/localstack/Makefile -v $$PACKAGES_DIR/moto:/opt/code/localstack/.venv/lib/python3.11/site-packages/moto/ -e TEST_PATH=\\'$(TEST_PATH)\\' -e LAMBDA_JAVA_OPTS=$(LAMBDA_JAVA_OPTS) $(ENTRYPOINT)" CMD="make test" make docker-run
 
-lint:              		  ## Run code linter to check code style
-	($(VENV_RUN); python -m ruff check --show-source .)
+lint:              		  ## Run code linter to check code style and check if formatter would make changes
+	($(VENV_RUN); python -m ruff check --show-source . && python -m black --check .)
```

Comment:
Assuming we have compatible `line-length` settings across `ruff` and `black`, this should be compatible: https://docs.astral.sh/ruff/faq/#is-the-ruff-linter-compatible-with-black

Reply:
If or once we decide to replace `black` as well, this won't be necessary anymore.

Reply:
Yeah, it is compatible, but ruff does not do all the formatting (for now, as you mentioned). So it's necessary to also run `black --check` until we switch to `ruff format`.